api/rebac/v1/lookup.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package api.rebac.v1;
+
+import "google/api/annotations.proto";
+import "rebac/v1/relationships.proto";
+
+option go_package = "ciam-rebac/api/rebac/v1;v1";
+option java_multiple_files = true;
+option java_package = "api.rebac.v1";
+
+service Lookup {
+    rpc Subjects(LookupSubjectsRequest) returns (stream LookupSubjectsResponse) {
```

Comment:
The problem is with the `stream ` response causing the problem with generating an HTTP client for the lookup service.

Remove the `stream` from the proto, run `make all` it will generate the http client for lookup. Add the stream afterwards

re-run `make all`